lib/msal-core/samples/react-sample-app/src/AuthProvider.js

```diff
@@ -117,12 +117,11 @@ export default C =>
         async componentDidMount() {
             msalApp.handleRedirectCallback(error => {
                 if (error) {
-                    // This currently doesn't work,
-                    // as the component mounts multiple times
-                    // and state gets lost.
-                    // this.setState({
-                    //     error: "Unable to acquire access token."
-                    // });
+                    const errorMessage = error && error.errorMessage ? error.errorMessage : "Unable to acquire access token.";
```

Comment:
This can be simplified, as line 119 already checks for the truthiness of `error`:

```js
const errorMessage = error.errorMessage || "Unable to acquire access token.";
```

Reply:
thanks for catching that!